cothink2/client/lib/helpers.js

```diff
@@ -0,0 +1,11 @@
+assert = function (cond, message) {
+    if (!cond) {
+        message = message || "Assertion failed.";
+        throw new Error(message);
+    }
+}
+
+assert_type = function (object, prototype) {
+    message = "Object " + object + " is not of type " + prototype + ".";
+    assert(prototype.isPrototypeOf(object), message);
+}
```

Comment:
Maybe use http://docs.meteor.com/#/full/check and/or http://docs.meteor.com/#/full/match_test